src/_guides/libraries/library-tour.md

```diff
@@ -2576,20 +2548,20 @@ The first parameter specifies the method to be invoked, and the second
 is a list of positional arguments to the method. An optional third
 parameter lets you specify named arguments.
 
-<!-- library-tour/mirrors/bin/main.dart -->
+<?code-excerpt "test/library_tour/mirrors_test.dart (invoke)"?>
 {% prettify dart %}
 var p = new Person('Bob', 'Smith', 42);
 InstanceMirror mirror = reflect(p);
 
-mirror.invoke(#greet, ['Shailen']);
+mirror.invoke(#greet, ['Sundar']);
```

Comment:
FYI, this change was copied from https://github.com/dart-lang/site-www/blob/792c517504acffcbffd07f85297fc43b134581ea/examples/library_tour/mirrors/bin/main.dart#L80-L81